akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/LatencySpec.scala

```diff
@@ -53,7 +42,7 @@ object LatencySpec extends MultiNodeConfig {
          }
          remote.artery {
            enabled = on
-           advanced.idle-cpu-level=8
+           advanced.idle-cpu-level=7
```

Comment:
If this is machine dependent, this could be a good thing to have as a self-optimizing parameter.